include/cantera/thermo/PengRobinson.h

```diff
@@ -0,0 +1,359 @@
+    virtual void getPartialMolarIntEnergies(double* ubar) const;
+    virtual void getPartialMolarCp(double* cpbar) const;
+    virtual void getPartialMolarVolumes(double* vbar) const;
+
+    //! Calculate the temperature dependent interaction parameter alpha needed for P-R EoS
+    /*
+    *  The temperature dependent parameter in P-R EoS is calculated as
+    *       \alpha = [1 + \kappa(1 - sqrt{T/T_crit}]^2
+    *  kappa is a function calulated based on the accentric factor.
+    *  Units: unitless
+	* a and b are species-specific coefficients used in P-R EoS, w is the acentric factor.
+    */
+    virtual void calculateAlpha(const std::string& species, double a, double b, double w);
+    //@}
+    /// @name Critical State Properties.
+    //@{
+
+    virtual double critTemperature() const;
+    virtual double critPressure() const;
+    virtual double critVolume() const;
+    virtual double critCompressibility() const;
+    virtual double critDensity() const;
+    virtual double speciesCritTemperature(double a, double b) const;
+
+public:
+    //@}
+    //! @name Initialization Methods - For Internal use
+    /*!
+     * The following methods are used in the process of constructing
+     * the phase and setting its parameters from a specification in an
+     * input file. They are not normally used in application programs.
+     * To see how they are used, see importPhase().
+     */
+    //@{
+
+    virtual bool addSpecies(shared_ptr<Species> spec);
+    virtual void initThermo();
+
+    //! Retrieve a and b coefficients by looking up tabulated critical parameters
+    /*!
+    *  If pureFluidParameters are not provided for any species in the phase,
+    *  consult the critical properties tabulated in data/inputs/critProperties.xml.
+    *  If the species is found there, calculate pure fluid parameters a_k and b_k as:
+    *  \f[ a_k = 0.4278*R**2*T_c^2/P_c \f]
+    *
+    *  and:
+    *  \f[ b_k = 0.08664*R*T_c/P_c \f]
+    *
+    *  @param iName    Name of the species
+    */
+    virtual std::vector<double> getCoeff(const std::string& iName);
+
+    //! Set the pure fluid interaction parameters for a species
+    /*!
+     *  The "a" parameter for species *i* in the Peng-Robinson model is assumed
+     *  to be a linear function of temperature:
+     *  \f[ a = a_0 + a_1 T \f]
+     *
+     *  @param species   Name of the species
+     *  @param a0        constant term in the expression for the "a" parameter
+     *      of the specified species [Pa-m^6/kmol^2]
+     *  @param a1        temperature-proportional term in the expression for the
+     *      "a" parameter of the specified species [Pa-m^6/kmol^2/K]
+     *  @param b         "b" parameter in the Peng-Robinson model [m^3/kmol]
+     *  @param alpha     dimensionless function of T_r and \omega
+     *  @param omega     acentric factor
+     */
+    void setSpeciesCoeffs(const std::string& species, double a0, double b,
+                              double w);

```

Comment:
This one should line up under the `const`.

Reply:
Corrected.